pkg/helm/charts/tests/securedclusterservices/testdata/helmtest/collector-config.test.yaml

```diff
@@ -0,0 +1,18 @@
+values:
```

Comment:
Maybe add another test for actually changing the values?

Reply:
I am not sure what you mean. Do you mean using different  values for `imagePullSecrets.allowNone`?

Reply:
No, sorry. I just attached the comment to L1 to indicate "in this test suite".

I think it would make sense to include a test which does not only flip the `enable` switch, but actually changes the values within the runtimeConfig and then verify that these values are properly propagated into the config map.

Reply:
I added a couple of unit tests, but I am not clear if they are what you wanted. In manual testing I have changed the values while upgrading. Initially I thought that is what you meant, but I don't think these tests are capable of doing the upgrade.